pkg/apis/fleet.cattle.io/v1alpha1/bundledeployment_types.go

```diff
@@ -407,6 +407,7 @@ type ModifiedStatus struct {
 	// +nullable
 	Name   string `json:"name,omitempty"`
 	Create bool   `json:"missing,omitempty"`
+	Exist  bool   `json:"exist,omitempty"`
```

Comment:
```suggestion
        // Exist is true if the modified resource exists in the cluster. This can happen, when ...
	Exist  bool   `json:"exist,omitempty"`
```

Reply:
> So this happens because we're missing annotations and the resource ends up in `plan.Create` instead of `plan.Update`?

It might not be accurate to say, that we're missing annotations. So far I was only able to reproduce the issue when two bundles have fought for ownership. When one has it and the other takes it, the first says "missing" instead of "isn't owned by me". This message is adapted in this PR.

> If we wrote our own `applied.DryRun`, could it return lists that fit our use case better? So that we don't have to do a an additional `Get` for every resource in `plan.Create`? I'm asking, because we do want to remove wrangler where possible.

Yes, I think we can do better when we write our own function to diff. However, I do not claim to have fully understood the diffing process.

And I only was able to reproduce the issue with two competing bundles, not with resources installed without a bundle and a bundle failing to properly adopt them. The newly create SURE might help to further investigate that.